charts/primary-site/values.yaml

```diff
@@ -79,6 +89,15 @@ streamService:
       namespace: ""
       subsystem: ""
     env: []
+    serviceAccount:
```

Comment:
Should this be per-service or is enabling service accounts something you'd do globally?

Reply:
We could technically use the inbox listener role everywhere as it has read/write access to both inbox and lake buckets, but I'd err on the side of least privilege and configure the services separately.